components/browser/icons/src/main/java/mozilla/components/browser/icons/processor/ResizingProcessor.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.icons.processor
+
+import android.content.Context
+import android.graphics.Bitmap
+import androidx.annotation.VisibleForTesting
+import mozilla.components.browser.icons.Icon
+import mozilla.components.browser.icons.IconRequest
+import mozilla.components.support.ktx.android.content.res.pxToDp
+
+/**
+ * [IconProcessor] implementation for resizing the loaded icon based on the target size.
+ */
+class ResizingProcessor(
```

Comment:
You haven't added this to the list of processors in `BrowserIcons` - was this intentional?

Reply:
This was intentional since you asked [not to introduce new icon behaviour](https://github.com/mozilla-mobile/android-components/pull/3263#pullrequestreview-246069317) previously.